sdk/ai/ai-projects/review/ai-projects.api.md

```diff
@@ -321,6 +334,11 @@ export interface CreateAndRunThreadOptions {
     truncation_strategy?: TruncationObject | null;
 }
 
+// Warning: (ae-forgotten-export) The symbol "CreateRunOptions_2" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export type CreateRunOptionalParams = Omit<CreateRunOptions_2, "assistant_id"> & OperationOptions;
```

Comment:
We usually call these just `CreateRunOptions` instead of `CreateRunOptionalParams` but this is probably not blocking for the first beta. It'll come up in GA API review though 😄

Reply:
thankfully our type names aren't constructors for params so we can pretty safely shuffle these around without impacting customers 👍

Reply:
The suffix Options already exists hence choose OptionalParams